package.json

```diff
@@ -47,7 +47,7 @@
     "eslint-plugin-sort-destructure-keys": "^1.3.5",
     "eslint-plugin-unicorn": "^20.1.0",
     "prettier": "^2.0.0",
-    "typescript": "^3.0.0"
+    "typescript": "^4.0.0"
```

Comment:
Maybe we should also update `@typescript-eslint/parser` and `@typescript-eslint/eslint-plugin`